tasklist/client/src/Tasks/Task/FormJS/index.tsx

```diff
@@ -183,37 +182,22 @@ const FormJS: React.FC<Props> = ({
               .otherwise(() => null)}
           </TaskDetailsRow>
           <DetailsFooter>
-            <AsyncActionButton
-              inlineLoadingProps={{
-                description: getCompletionButtonDescription(submissionState),
-                'aria-live': ['error', 'finished'].includes(submissionState)
-                  ? 'assertive'
-                  : 'polite',
-                onSuccess: () => {
-                  onSubmitSuccess();
-                  setSubmissionState('inactive');
-                },
+            <CompleteTaskButton
+              submissionState={submissionState}
+              onClick={() => {
+                setSubmissionState('active');
+                formManagerRef.current?.submit();
               }}
-              buttonProps={{
-                className: taskState === 'COMPLETED' ? 'hide' : '',
-                size: 'md',
-                type: 'submit',
-                disabled: submissionState === 'active' || !canCompleteTask,
-                onClick: () => {
-                  setSubmissionState('active');
-                  formManagerRef.current?.submit();
-                },
-                title: canCompleteTask
-                  ? undefined
-                  : 'You must first assign this task to complete it',
+              onSuccess={() => {
+                onSubmitSuccess();
+                setSubmissionState('inactive');
               }}
-              status={submissionState}
               onError={() => {
                 setSubmissionState('inactive');
               }}
-            >
-              Complete Task
-            </AsyncActionButton>
+              isHidden={taskState === 'COMPLETED'}
+              disabled={!canCompleteTask}
```

Comment:
Just to be consistent with the rest

```suggestion
              isDisabled={!canCompleteTask}
```